RBDM/Debug/RBDM.Build.CppClean.log

```diff
@@ -0,0 +1,18 @@
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\vc120.pdb
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\vc120.idb
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\base.obj
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\double.obj
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\integer.obj
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\main.obj
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\record.obj
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\string.obj
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\table.obj
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\rbdm.ilk
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\rbdm.exe
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\rbdm.pdb
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\rbdm.tlog\cl.command.1.tlog
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\rbdm.tlog\cl.read.1.tlog
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\rbdm.tlog\cl.write.1.tlog
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\rbdm.tlog\link.command.1.tlog
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\rbdm.tlog\link.read.1.tlog
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\rbdm.tlog\link.write.1.tlog
```

Comment:
you should commit only project's changes, no build results, no binary files. Remove all exe and rest binary files from here